src/views/Dashboard.js

```diff
@@ -171,14 +169,6 @@ const Dashboard = () => {
           <Box className={classes.migrationContainer}>
             <Typography align="center" variant="body1">
               {translate('Login.migrationText1')}
-              <br></br>
-              <ExternalLink
-                classes={{ root: 'body3_link_gradient' }}
-                href={MIGRATION_INFO_URL}
-                variant="body3"
```

Comment:
shouldn't it be "body4"?

Reply:
it is body3 but anyway this ExternalLink gets deleted